### PR TITLE
blog: Add 6-month Discourse analysis

### PR DESCRIPTION
**Pull request by GregSutcliffe**
Here's a little analysis I've been cooking up to celebrate the 6 month anniversary of our move to Discourse :)

### CODE REVIEW

File: _posts/2018-07-26-discourse-6-months-on-impact-assesment.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+date: "2018-07-26"
+output:
+tags:
+- foreman
+- discourse
+- data-science
+---
+
+It's been 6 months since the Foreman community switched to Discourse, so I
+wanted to revisit some of the graphs I [drew in November][forum_post] (when I
```
Comment (ares) [inline _posts/2018-07-26-discourse-6-months-on-impact-assesment.md]: double [] and space before (), the link does not render

**Review by ares — CHANGES_REQUESTED** [review]
other than one nit looks good, the results made my day :-)

File: _posts/2018-07-26-discourse-6-months-on-impact-assesment.md (hunk 1 above)
Comment (GregSutcliffe) [reply _posts/2018-07-26-discourse-6-months-on-impact-assesment.md]: Thanks @ares fixed

**Review by ares — APPROVED** [review]

**Comment by ares** [comment]
Thanks @GregSutcliffe, merging.

**Pull request merged**